### DISCUSSION TITLE
Performance: Have we considered trying to use DataParallel? It looks promising for people with more than one GPU

### DISCUSSION THREAD

**Discussion by ilikenwf** [vladmandic/sdnext#1482] [General]
Consider it could be interesting in cases where vram is limited...

https://pytorch.org/tutorials/beginner/blitz/data_parallel_tutorial.html

**Comment by vladmandic** [comment]
sounds interesting, but i'd need multiple gpus to do anything :)

**Comment by ilikenwf** [comment]
I'm in the same boat...it appears to work regardless of how many gpus you actually have...

I suspect it is aimed more at situations where you have several similar GPUs but would likely work even if you had a mix of nvidias...I have just a 1080ti 11gb personally and while it does still get the job done I wouldn't mind the benefit of more VRAM and more processing power...

**Reply by Tillerz** [reply]
And I think if you add a 4090 to that, the benefit of shelling out stuff to the 1080 as a secondary device will be minimal. You'd also need a PC that is big enough to get it all in and a mainboard that has two slots far enough apart and a powersource that can handle all of it, too. :P
